gax-java/gax/src/main/java/com/google/api/gax/core/GaxProperties.java

```diff
@@ -100,6 +100,23 @@ public static String getGaxVersion() {
 
   /** Returns the current runtime version */
   private static String getRuntimeVersion() {
-    return System.getProperty("java.version");
+    String javaVersion = System.getProperty("java.version");
+    String combined = javaVersion;
+
+    // append the vendor information to the java-version if vendor is present.
+    String vendor = System.getProperty("java.vendor");
+    if (vendor != null && !vendor.isEmpty()) {
```

Comment:
This can be simplified by Guava's `Strings.isNullOrEmpty`, similar to [here](https://github.com/googleapis/sdk-platform-java/blob/6e9ef3f157957a57d5e1c06cee7e755054db6af0/gapic-generator-java/src/main/java/com/google/api/generator/gapic/model/Service.java#L52)